M2/Macaulay2/d/CMakeLists.txt

```
@@ -135,7 +135,8 @@ endforeach()
 
 # To avoid unnecessary recompiling, only version.dd is given GIT_DESCRIPTION
 set_source_files_properties(version-tmp.cc
-  PROPERTIES COMPILE_DEFINITIONS GIT_DESCRIPTION="${GIT_DESCRIPTION}")
+  PROPERTIES COMPILE_DEFINITIONS
+  "GIT_DESCRIPTION=\"${GIT_DESCRIPTION}\";GIT_BRANCH=\"${GIT_BRANCH}\"")
```

This is a little awkward, I would prefer:
```suggestion
set_property(SOURCE version-tmp.cc APPEND PROPERTY COMPILE_DEFINITIONS
  GIT_DESCRIPTION="${GIT_DESCRIPTION}"
  GIT_BRANCH="${GIT_BRANCH}")
```

---

Thanks for the suggestion!  Applied in https://github.com/Macaulay2/M2/pull/3247/commits/b4236cfeb488f4fe908bc4b179c8d8433f8df4d9